conformance/utils/suite/suite.go

```diff
@@ -177,6 +186,12 @@ func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
 		}
 	}
 
+	for _, feature := range test.MinChannel {
+		if test.MinChannel < suite.MinChannel {
```

Comment:
Although this works since "experimental" is < "standard", I think it would be better to actually use an enum here in case we ever add a "ga_only" channel or something that doesn't work that way. So in this case a new "Channel" type with something like this:

```golang
const (
	Experimental Channel = 1
	Standard Channel = 2
)
```